### DISCUSSION TITLE
Pls help with my folder change.

### DISCUSSION THREAD

**Discussion by philboycn** [transmission/transmission#1948] [Q&A] [answered]
hello , i have a huge number of downloaded torrents, they all have different folder name. i want to change the downloaded folder like this  : "/path/to/ABC/ABC" to "/path/to2/ABC/ABC". is there a way can do it not one by one? Pls help!

**Comment by sio** [comment, marked as answer]
You can write a script that calls transmission-remote or you can call JSON RPC from any programming language you're familiar with.
